AspNetCoreAnalyzers.Vsix/AspNetCoreAnalyzers.Vsix.csproj

```diff
@@ -7,8 +7,16 @@
     <RootNamespace>AspNetCoreAnalyzers</RootNamespace>
     <TargetName>AspNetCoreAnalyzers</TargetName>
     <NoWarn>NU1605</NoWarn>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
```

Comment:
This repository uses paket to manage dependencies hence the build error on CI. This is not a problem for merging, I can merge it and fix after np.